src/page-templates/content-page/content-page-template.tsx

```diff
@@ -458,7 +458,7 @@ const ContentPageTemplate: NextPage<ContentPageProps> = ({
                                 src={getPlaceHolderImage(image?.url)}
                                 sx={{
                                     borderRadius: 'inc30',
-                                    objectFit: 'cover',
+                                    objectFit: 'fill',
```

Comment:
I'd say maybe just do this for events for now. This is having some stretching effects in existing articles 
<img width="903" alt="Screen Shot 2023-01-19 at 12 34 18 PM" src="https://user-images.githubusercontent.com/46664803/213518731-7a08ebe1-ec94-4ad9-9a21-96c31497e139.png">
<img width="831" alt="Screen Shot 2023-01-19 at 12 34 44 PM" src="https://user-images.githubusercontent.com/46664803/213518745-59b3659f-0c7e-4332-b430-f42d014c570e.png">

Reply:
@mcolella14  ah damn, checked out a bunch of different content types but didn't see these. FWIW though we need to solve for this outside of events. A happy medium I found was to use `objectFit: contain`, it won't fill the containers width if the image isn't that wide though (see screenshots), which I assume Mike will not be stoked about. Will keep looking to see if I can figure out a quick simple solution

Reply:
Ahh ok yeah I think that works for me, but yeah unless we get the content writers to size their images correctly i think something is always gonna be off, but I think `contain` is a good medium for now!

Reply:
@mcolella14 reverting the change after speaking with @harikakotipalli. She brought up the point that the user should adjust the image asset instead of us making a code change, since this hasn't been brought up in the last ~8 months of being live.